deployer/scripts/upgrade.sh

```diff
@@ -754,6 +772,9 @@ function upgrade_logging() {
         6)
           common_data_model=true
           ;;
+        7)
```

Comment:
to detect that this is the current version we'll also need to update the variable in the Dockerfile and `getDeploymentVersion`